Can't run tests on windows

### Reproduction link
[https://stackblitz.com/sorry-there-is-no-link](https://stackblitz.com/sorry-there-is-no-link)

### Steps to reproduce
1. Clone
2. npm install
3. npm test

### What is expected?
Expected to run tests

### What is actually happening?
```
> ng-zorro-antd@0.0.0-NOT-USED test C:\Users\******\Repos\ng-zorro-antd
> ng test --no-watch --no-progress --browsers=ChromeHeadlessCI --code-coverage

21 01 2020 08:25:14.558:INFO [karma-server]: Karma v4.3.0 server started at http://0.0.0.0:9877/
21 01 2020 08:25:14.569:INFO [launcher]: Launching browsers ChromeHeadlessCI with concurrency unlimited
21 01 2020 08:25:14.684:INFO [launcher]: Starting browser ChromeHeadless
21 01 2020 08:26:14.686:WARN [launcher]: ChromeHeadless have not captured in 60000 ms, killing.
21 01 2020 08:26:16.688:WARN [launcher]: ChromeHeadless was not killed in 2000 ms, sending SIGKILL.
21 01 2020 08:26:18.690:WARN [launcher]: ChromeHeadless was not killed by SIGKILL in 2000 ms, continuing.

npm ERR! Test failed.  See above for more details.
```
| Environment | Info |
|---|---|
| ng-zorro-antd | 8.5.2 |
| Browser | - |

---
Environment:
Windows 10
node v10.15.0
npm 6.13.1

I've deleted `node-modules` and reinstalled it and tests failed again.
I googled and couldn't find any solution.<!-- generated by ng-zorro-issue-helper. DO NOT REMOVE -->

More on Windows:

`npm run test:watch` fails. Showing:

```log
❯ npm run test:watch rate

> ng-zorro-antd@0.0.0-NOT-USED test:watch C:\Users\wendellhu\Developer\OSS\ng-zorro-antd
> gulp test:watch --tags $tags "rate"

[18:21:12] Using gulpfile ~\Developer\OSS\ng-zorro-antd\gulpfile.js
[18:21:12] Task never defined: rate
[18:21:12] To list available tasks, try running: gulp --tasks
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ng-zorro-antd@0.0.0-NOT-USED test:watch: `gulp test:watch --tags $tags "rate"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ng-zorro-antd@0.0.0-NOT-USED test:watch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\wendellhu\AppData\Roaming\npm-cache\_logs\2020-01-22T10_21_13_005Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
   "scripts": {
     "start": "gulp start:dev",
     "test": "ng test --no-watch --no-progress --browsers=ChromeHeadlessCI --code-coverage",
-    "test:watch": "gulp test:watch --tags $tags",
+    "test:watch": "gulp test:watch --tags",
     "test:schematics": "gulp build:schematics && gulp test:schematics",
     "build": "gulp build:release",
     "build:lib": "gulp build:library",

--- a/scripts/gulp/util/task-helpers.ts
+++ b/scripts/gulp/util/task-helpers.ts
@@ -16,7 +16,7 @@ export function execTask(binPath: string, args: string[], env: {} = {}): gulp.Ta
     (process.stdout as any)._handle.setBlocking(true);
     // tslint:disable-next-line:no-any
     (process.stdout as any)._handle.setBlocking(true);
-    const bin = os.platform() === 'win32' ? `${binPath}.cmd` : binPath;
+    const bin = os.platform() === 'win32' && binPath === 'ng' ? `${binPath}.cmd` : binPath;
     const childProcess = child_process.spawn(bin, args, {
       env: { ...process.env, ...env },
       cwd: process.cwd(),

--- a/scripts/gulp/util/task-helpers.ts
+++ b/scripts/gulp/util/task-helpers.ts
@@ -1,5 +1,6 @@
 import * as child_process from 'child_process';
 import * as gulp from 'gulp';
+import * as os from 'os';
 
 const gulpClean = require('gulp-clean');
 const resolveBin = require('resolve-bin');
@@ -15,7 +16,8 @@ export function execTask(binPath: string, args: string[], env: {} = {}): gulp.Ta
     (process.stdout as any)._handle.setBlocking(true);
     // tslint:disable-next-line:no-any
     (process.stdout as any)._handle.setBlocking(true);
-    const childProcess = child_process.spawn(binPath, args, {
+    const bin = os.platform() === 'win32' ? `${binPath}.cmd` : binPath;
+    const childProcess = child_process.spawn(bin, args, {
       env: { ...process.env, ...env },
       cwd: process.cwd(),
       stdio: 'inherit'

